src/Tizen.Applications.Common/Tizen.Applications.RPCPort/Parcel.cs

```diff
@@ -18,6 +18,135 @@
 
 namespace Tizen.Applications.RPCPort
 {
+    /// <summary>
+    /// This structure represents the time stamp.
+    /// </summary>
+    /// <since_tizen> 9 </since_tizen>
+    public class TimeStamp
+    {
+        /// <summary>
+        /// Constructor with TimeStamp.
+        /// </summary>
+        /// <exception cref="InvalidIOException">Thrown when an internal IO error occurs.</exception>
+        /// <since_tizen> 9 </since_tizen>
+        internal TimeStamp(Interop.LibRPCPort.Parcel.TimeStamp time)
+        {
+            second = time.sec;
+            nanoSecond = time.nsec;
+        }
+
+        /// <summary>
+        /// The second of TimeStamp.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public long second
```

Comment:
PascalCasing을 사용하는것이 룰입니다.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers

```suggestion
        public long Second
```